packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/linkedin_source
-    version: [">=0.10.0", "<0.11.0"]
\ No newline at end of file
+  # - package: fivetran/linkedin_source
+  #   version: [">=0.11.0", "<0.12.0"]
+  - git: "https://github.com/fivetran/dbt_linkedin_source"
```

Comment:
obligatory reminder to update before merging

Reply:
same